common/src/Renderer/PrimitiveRenderer.cpp

```diff
@@ -367,11 +367,12 @@ void PrimitiveRenderer::renderLines(RenderContext& renderContext)
 {
   ActiveShader shader(renderContext.shaderManager(), Shaders::VaryingPUniformCShader);
 
+  const auto scaledLineWidth = static_cast<float>(renderContext.dpiScale());
```

Comment:
This variable name is wrong. I would also suggest not to cast here and to store `dpiScale` as `float` in `RenderContext`.